models/attribute/src/transformers.ts

```diff
@@ -0,0 +1,23 @@
+import { Transformer } from '@commercetools-test-data/core';
+import type { TAttribute, TAttributeDefault, TAttributeGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TAttributeDefault, TAttributeDefault>('default', {
+    buildFields: ['attributeDefinition'],
+  }),
+  rest: Transformer<TAttributeDefault, TAttribute>('rest', {
+    buildFields: [],
+    replaceFields: ({ fields }) => ({
+      name: fields.name,
+      value: fields.value,
+    }),
+  }),
+  graphql: Transformer<TAttributeDefault, TAttributeGraphql>('graphql', {
+    buildFields: ['attributeDefinition'],
+    addFields: () => ({
+      __typename: 'RawProductAttribute',
```

Comment:
So this is definitively the type used by `ProductVariant` in GraphQL, but the draft version `ProductVariantInput` uses `ProductAttributeInput` and the fields are different. Really not sure what to do here.

![2023-02-16 14 43 38](https://user-images.githubusercontent.com/8136478/219471107-e760080a-edd9-42cf-92c6-2d3ca324c109.gif)

Reply:
Hmm...interesting.

I'm unsure if RawProductAttribute and ProductAttributeInput have a model/draft relationship. If they do, it makes sense to pull ProductAttributeInput in as sort of a faux-draft, but again not sure.

Worst case scenario is to create ProductAttributeInput separately, and refactor as needed once we get in some more questions with Nicola 🤷‍♂️ Does that seem appropriate?

Reply:
I feel like `ProductAttributeInput` is more similar to the REST `Attribute` model and `RawProductAttribute` is a weird GraphQL thing

Reply:
But also, `RawProductAttribute` is the model and `ProductAttributeInput` is the draft in GraphQL land

Reply:
<img width="1257" alt="image" src="https://user-images.githubusercontent.com/15024562/219476534-7f4b3350-cd9c-4539-b69e-7958bd95c875.png">

So I suppose we do need a draft here, but _only_ for the graphql side of things, because `ProductAttributeInput` is (for graphql land) a draft.

Something I'll touch base with Nicola on, we haven't really encountered this before iirc!

Reply:
![image](https://user-images.githubusercontent.com/15024562/220937970-5e14e423-9100-4a76-b5fb-bb74bf0cea3a.png)

Resolved: we will add a "dummy" `AttributeDraft` that will generate the appropriate graphql transformation to `ProductVariantInputAttributes`. This is for consistency with existing test-data repo structure.

Reply:
@stephsprinkle @tylermorrisford [This commit](https://github.com/commercetools/test-data/pull/203/commits/1a8a96a3650b772b53004d14b4ddf471b1161573) addresses these findings.